hlx_statics/components/code.js

```diff
@@ -6,6 +6,7 @@ export default function decoratePreformattedCode({ block, language }) {
   pre.classList.add('line-numbers');
 
   const code = block.querySelector('code');
+  code.textContent = code.textContent.trim();
```

Comment:
Should we trim? I think if user starts (or ends) their code with a bunch of spaces, we should leave those in.

```suggestion
```

Reply:
@melissag-ensemble , If we remove this, it would add an extra line to the code. That’s why I used it. However, it’s okay, we will remove it. I’ve attached the screenshot below for your reference.
![image](https://github.com/user-attachments/assets/94001abc-10f5-453a-9929-bd9372a311a1)

Reply:
Nice work, @melissag-ensemble !!!

I do have one question:
![image](https://github.com/user-attachments/assets/29896359-6f4e-469d-bcd2-f892e40e8d58)

In every code block, an extra line gets added—even if it contains a single line. How can we handle this issue?

Reply:
Oh you're right... sorry I missed your comment. I think we need to fix this before it gets to `code.js` because ideally we shouldn't be removing whitespace intended by the user. 

I feel like somewhere along the way when the markdown gets converted to markup, an extra new line gets added somewhere..